data/json/items/ammo.json

```diff
@@ -1440,7 +1440,7 @@
         "name_plural" : "5.56 NATO SS109",
         "symbol" : "=",
         "color" : "yellow",
-        "description" : "5.56x45mm ammunition with a 62gr FMJ bullet containing a steel penetrator. It's light weight, high accuracy and managable recoil lead to it's use by NATO countries and many more. The round's yaw and fragmentation cause significant damage inside a target.",
```

Comment:
I've fixed only single spaces, but this description still need some love (_It's_ vs. _Its_ fix and general editor's work are required).